changelog/8801.improvement.md

```diff
@@ -0,0 +1 @@
+Change the confidence score assigned by [`FallbackClassifier`](https://rasa.com/docs/rasa/components#fallbackclassifier)  to fallback intent to be the same as the fallback threshold.
```

Comment:
```suggestion
Change the confidence score assigned by [`FallbackClassifier`](components.mdx#fallbackclassifier) to fallback intent to be the same as the fallback threshold.
```

Reply:
On this one, shouldn't it actually be `/docs/docs/components.mdx#fallbackclassifier`? Without the full path, PyCharm complains `Cannot resolve file 'components.mdx'`. Even so, I still get a warning about `Cannot resolve anchor #fallbackclassifier` 🤔

Reply:
Mhm, I'm pretty sure that this should work. When you run `make livedocs` - does it work?

Reply:
Yup, it worked, so all safe to make the change you suggested?

Reply:
yep 👍🏻